packs/feats/agile-hand.json

```
@@ -11,7 +11,7 @@
         },
         "category": "class",
         "description": {
-            "value": "<p>When you cast mage hand, you can modify its target to be a set of thieves' tools. When you Cast the Spell in this way, the tools move up to 20 feet towards a device or lock. After you Sustain the Spell on future turns, you can use the tools to attempt Thievery checks to Disable a Device or Pick a Lock within the spell's range, but you take a -2 penalty to your Thievery check. If you critically fail, the spell ends and you can't use this modification again for 24 hours.</p>"
+            "value": "<p>When you cast @UUID[Compendium.pf2e.spells-srd.Item.Telekinetic Hand], you can modify its target to be a @UUID[Compendium.pf2e.equipment-srd.Item.Thieves' Toolkit]. When you Cast the Spell in this way, the tools move up to 20 feet towards a device or lock. After you Sustain the spell on future turns, you can use the tools to attempt Thievery checks to @UUID[Compendium.pf2e.actionspf2e.cYdz2grcOcRt4jk6]{Disable a Device} or @UUID[Compendium.pf2e.actionspf2e.2EE4aF4SZpYf0R6H]{Pick a Lock} within the spell's range, but you take a -2 penalty to your Thievery check. If you critically fail, the spell ends and you can't use this modification again for 24 hours.</p>"
```

```suggestion
            "value": "<p>When you cast @UUID[Compendium.pf2e.spells-srd.Item.Telekinetic Hand], you can modify its target to be a @UUID[Compendium.pf2e.equipment-srd.Item.Thieves' Toolkit]. When you Cast the Spell in this way, the tools move up to 20 feet towards a device or lock. After you Sustain the spell on future turns, you can use the tools to attempt Thievery checks to @UUID[Compendium.pf2e.actionspf2e.Item.Disable a Device] or @UUID[Compendium.pf2e.actionspf2e.Item.Pick a Lock] within the spell's range, but you take a -2 penalty to your Thievery check. If you critically fail, the spell ends and you can't use this modification again for 24 hours.</p>"
```
Are you using quick insert? If so highlight the text, search, then drag the result into the text window. That will make Foundry make the link and extract will reformat it.

---

I am. I'm also aware of this, but I think I must be clicking instead of dragging some by accident and failed to catch them when reviewing. I'll be more careful.